cypress/integration/lab.test.js

```diff
@@ -0,0 +1,45 @@
+const deckId = "my-deck-id";
+
+const clickForDeck = (fixture, count = 5) => {
+  cy.intercept(`https://deckofcardsapi.com/api/deck/${deckId}/draw/?count=${count}`, { fixture });
+  cy.get("button").click();
+};
+
+const visitWithFirstDeck = () => {
+  cy.intercept("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1", {
+    deck_id: deckId,
+  }).as("newDeck");
+
+  cy.visit("./index.html");
+
+  clickForDeck("cards1.json");
+};
+
+describe("Index", () => {
+  it("starts the select with a default value of 5", () => {
+    cy.visit("./index.html");
+    cy.get("select").should("have.value", 5);
+  });
+
+  it("shows five cards from the retrieved deck ID when the button is clicked", () => {
+    visitWithFirstDeck();
+
+    cy.get(".card")
+      .should("have.length", 5)
+      .first()
+      .should("have.attr", "src", "https://deckofcardsapi.com/static/img/2D.png");
+  });
```

Comment:
A couple of things might slip through the cracks here that I think are worth testing (and are doable with the fixtures).
- When we fetch 5 cards, all 3 cards are the expected cards from the fixtures (not the same single card repeated 5 times)
- When we fetch 3 cards next time around, all 3 cards are the expected cards from the fixtures (not reused or repeated)